playground/vue/pre-compiled/external-cssmodules.vue.js

```diff
@@ -0,0 +1,59 @@
+import {
+  Fragment as _Fragment,
```

Comment:
Excuse me. Can I ask why the imported variable has a new name starting with an underscore?

Reply:
These are imports generated by  `@vue/compiler-core`
https://github.com/vuejs/core/commit/51317af6e8405490c4e17694cc3c767ecd58b3d7

It's to avoid the runtime helpers' names colliding with user-defined variables in the same scope.
For example, the `renderList` helper, generated by compiling `v-for`, if not renamed, has a high chance of collision with a user-defined variable: https://github.com/search?q=+renderList+language%3AVue&type=code

Reply:
Thanks for your reply!